apps/pie-docs/src/assets/styles/base/_page.scss

```
@@ -18,3 +18,10 @@ body {
         scroll-behavior: auto !important;
     }
 }
+
+/* CSS rules to be applied when running visual regression test */
+@media only percy {
+    .hide-in-percy {
+        visibility: hidden;
+    }
+}
```

I wonder if we could do something like this?
It _should_ switch the date for `XX/XX/XXXX`, so it's still present in the screenshot. That way, if anything else (like font-size, position, etc) changes or it gets deleted then the screenshot should still show a change.

```suggestion
@media only percy {
    .updated-date {
        visibility: hidden;
    
        &:before {
            visibility: visible;
            content: "XX/XX/XXXX";
        }
    }
}
```

---

Ah yeah, good shout!